examples/flutter_gallery/lib/demo/cupertino/cupertino_activity_indicator_demo.dart

```diff
@@ -0,0 +1,22 @@
+// Copyright 2017 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+
+class CupertinoProgressIndicatorDemo extends StatelessWidget {
+  static const String routeName = '/cupertino_progress_indicator';
```

Comment:
Maybe `/cupertino/progress_indicator` ?  These names don't really matter, but might as well be pretty.